ci/playbooks/edpm_build_images/edpm_build_images_content_provider_run.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- hosts: all
+  gather_facts: true
+  tasks:
+    - name: Build edpm images
+      ansible.builtin.command:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+        cmd: >-
+          ansible-playbook
+          ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml
+          -e @scenarios/centos-9/base.yml
+          {%- if cifmw_extras is defined %}
+          {%-   for extra_vars in cifmw_extras %}
+          -e "{{   extra_vars }}"
+          {%-   endfor %}
+          {%- endif %}
+          -e @scenarios/centos-9/zuul_inventory.yml
+          -e @scenarios/centos-9/edpm_build_images_content_provider.yml
+          -e "cifmw_rp_registry_ip={{ cifmw_rp_registry_ip }}"
+
+    - name: Include inner ansible vars file
+      ansible.builtin.slurp:
+        src: "{{ cifmw_artifacts_basedir }}/artifacts/ansible-vars.yml"
+      register: inner_ansible
+
+    - name: Get inner ansible vars
+      ansible.builtin.set_fact:
+        inner_ansible_vars: "{{ inner_ansible.content | b64decode | from_yaml }}"
+
+    - name: Set content provider
+      ansible.builtin.set_fact:
```

Comment:
cool ... had to go check that set_fact can take a list of things at once 
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/set_fact_module.html#parameter-key_value